mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -551,7 +551,13 @@ def LLVM_InvokeOp : LLVM_Op<"invoke", [
                    Variadic<LLVM_Type>:$normalDestOperands,
                    Variadic<LLVM_Type>:$unwindDestOperands,
                    OptionalAttr<DenseI32ArrayAttr>:$branch_weights,
-                   DefaultValuedAttr<CConv, "CConv::C">:$CConv);
+                   DefaultValuedAttr<CConv, "CConv::C">:$CConv,
+                   VariadicOfVariadic<LLVM_Type,
+                                      "op_bundle_sizes">:$op_bundle_operands,
+                   DenseI32ArrayAttr:$op_bundle_sizes,
+                   OptionalProperty<
+                     ArrayProperty<StringProperty, "operand bundle tags">
+                   >:$op_bundle_tags);
```

Comment:
```suggestion
                     ArrayProperty<StringProperty, "operand bundle tags">:$op_bundle_tags);
```
Not sure I understand why we need to wrap this in an optional. Is there a semantic difference between an empty array and the array being non-present?

Reply:
Because `LLVM_CallIntrinsicOp` uses declarative assembly format and `OptionalProperty` could prevent `op_bundle_tags` from appearing in the assembly if it's non-present. Anyway as we're going to change the assembly syntax, I'll create a custom printer and parser for `LLVM_CallIntrinsicOp` and the `OptionalProperty` could be removed then.